packages/integration/src/github/SingleInstanceGithubCredentialsProvider.ts

```diff
@@ -96,9 +96,7 @@ class GithubAppManager {
     const { installationId, suspended } = await this.getInstallationData(owner);
     if (this.allowedInstallationOwners) {
       if (!this.allowedInstallationOwners?.includes(owner)) {
-        throw new Error(
-          `The GitHub application for ${owner} is not included in the allowed installation list (${installationId}).`,
-        );
+        return { accessToken: '' }; // An empty token allows anonymous access to public repos
```

Comment:
How tricky is it to wire this up to return `accessToken: undefined` instead? Feel it's unfortunate to hide what's going on here from the type checking

Reply:
Trivial, it seems.